README.md

```diff
@@ -1,173 +1,246 @@
 
-This mode is recommended to quickly test the operator during its development:
+![Topology](./docs/assets/topology-main.png)
 
-```
-operator-sdk run bundle $BUNDLE_IMG
-```
+These dashboards are accessible directly from the main menu, and also as contextual tabs for any Pod, Deployment, Service (etc.) in their details page.
 
-### Deploy as bundle from the Console's OperatorHub page
+![Contextual topology](./docs/assets/topology-deployment.png)
 
-This mode is recommended when you want to test the customer experience of navigating through the
-operators' catalog and installing/configuring it manually through the UI.
+### Standalone console
 
-First, create and push a catalog image:
+_Coming soon_
 
+### Grafana
+
+Grafana can be used to retrieve and show the collected flows from Loki. If you used the `make` commands provided above to install NetObserv from the repository, you should already have Grafana installed and configured.
+
+Else you can [find here](https://github.com/netobserv/documents/blob/main/hack_loki.md#grafana) some help to install Grafana if needed.
+
+Then import [this dashboard](./config/samples/dashboards/Network%20Observability.json) in Grafana. It includes a table of the flows and some graphs showing the volumetry per source or destination namespaces or workload:
+
+![Grafana dashboard](./docs/assets/netobserv-grafana-dashboard.png)
+
+## Configuration
+
+The `FlowCollector` resource is used to configure the operator and its managed components. A comprehensive documentation is [available here](./docs/FlowCollector.md), and a full sample file [there](./config/samples/flows_v1alpha1_flowcollector.yaml).
+
+To edit configuration in cluster, run:
+
+```bash
+kubectl edit flowcollector cluster
 ```
-export CATALOG_IMG=quay.io/$USER/network-observability-operator-catalog:v$VERSION
-make catalog-build catalog-push catalog-deploy
-```
 
-The NetObserv Operator is available in OperatorHub: https://operatorhub.io/operator/netobserv-operator
+As it operates cluster-wide, only a single `FlowCollector` is needed and possible, and it has to be named `cluster`.
+
+A couple of settings deserve special attention:
+
+- Agent (`spec.agent`) can be `ipfix` or `ebpf`. As mentioned above, the IPFIX option is fully functional when using OVN-Kubernetes CNI (other CNIs are not supported, but you may still be able to configure them manually if they allow IPFIX exports) whereas eBPF is expected to work regardless the running CNI.
+
+- Sampling (`spec.ipfix.sampling` and `spec.ebpf.sampling`): 24/7 unsampled flow collection may consume a non-negligible amount of resources. While we are doing our best to make it a viable option in production, it is still often necessary to mitigate by setting a sampling ratio. A value of `100` means: one flow every 100 is sampled. `1` means no sampling. The lower it is, the more accurate are flows and derived metrics.
```

Comment:
Yes, it was on purpose. Even if it can be vastly improved, we considered the performance as "good enough". I'd keep it as disabled, considering that eBPF right now is not the default agent.

Second question: yes, we assumed it would default to 0. We can set it explicitly to 0 or 1 if you prefer to make it clearer. Or to another value if you consider that we should enable sampling by default.

Reply:
Fair enough, let's keep it that way :)